.circleci/config.yml

```diff
@@ -195,6 +188,15 @@ jobs:
           command: ./scripts/deploy.sh
 
 commands:
+  install-golang:
+    steps:
```

Comment:
If we’re making this a command, we should make the go version a parameter to make its behaviour clearer

Reply:
good idea - done.